development-docs/systemtests/io.strimzi.systemtest.cruisecontrol.CruiseControlApiST.md

```diff
@@ -0,0 +1,52 @@
+# CruiseControlApiST
+
+**Description:** This test suite verifies that Cruise Control's basic API requests function correctly
+
+**Before tests execution steps:**
+
+| Step | Action | Result |
+| - | - | - |
+| 1. | Deploy the cluster operator | Cluster operator is deployed |
+
+**Labels:**
+
+* [cruise-control](labels/cruise-control.md)
+
+<hr style="border:1px solid">
+
+## testCruiseControlAPIUsers
+
+**Description:** This test case verifies the creation and usage of CruiseControl's API users.
+
+**Steps:**
+
+| Step | Action | Result |
+| - | - | - |
+| 1. | Create broker and controller KafkaNodePools. | Both KafkaNodePools are successfully created |
```

Comment:
some description have periods at the end, most not. I'd have all without I think

Reply:
Udpated